normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -68,7 +68,20 @@ let Normandy = {
 
     client() {
         return new Promise(resolve => {
-            let client = {};
+            let client = {
+                plugins: {}
+            };
+
+            // Populate plugin info.
+            for (let k = 0; k < navigator.plugins.length; k++) {
+                let plugin = navigator.plugins[k];
```

Comment:
Because navigator.plugins isn't actually an Array, it's a PluginArray. But I didn't actually test if that still works.

I could've used `Array.from`, but I somehow thought that `Array.from` wasn't available in the Firefox versions we care about, but looking at the docs, I was wrong. I'll see about fixing this up.